app/views/pages/home.html.erb

```diff
@@ -4,7 +4,7 @@
 <%# Hero %>
 
 <section class="home">
-  <section id='startpage__start-teaser<%= "_#{rand(5)}" unless current_region == :ooe %>' class="start-teaser startpage__photo-bar">
+  <section id='startpage__start-teaser<%= "_#{[0,2,3,4].sample}" unless current_region == :ooe %>' class="start-teaser startpage__photo-bar">
```

Comment:
Could we also please remove the image for the moment since it is not the final version. Let's load it up again once it is finished.